README.md

```diff
@@ -1,11 +1,169 @@
-# Peggy
+# ETH Bridge Zone
 
-Peggy is a peg zone implementation that allows the movement of tokens and 
-potentially other assets from Ethereum (or any EVM) to Cosmos (or any
-Tendermint) chain. 
+[![CircleCI](https://circleci.com/gh/cosmos/cosmos-ethereum-bridge/tree/master.svg?style=svg)](https://circleci.com/gh/cosmos/cosmos-ethereum-bridge/tree/master)
```

Comment:
this should be `cosmos/peggy` right?

Reply:
Likely yes, though this will need to be confirmed and setup by whoever has access to the cosmos circleci account. have fixed it here: https://github.com/cosmos/peggy/pull/51/commits/3afbbe1e2c705fea5699d5720a402fbb302f3a42